src/alexapi/playback_handlers/vlchandler.py

```diff
@@ -34,27 +34,32 @@ def on_setup(self):
 		]
 
 		parametersSpeech = parametersCommon
+		
+		if not self.__config['platform']['device'] == 'windows':
```

Comment:
Detect platform using the `platform` module ... see https://stackoverflow.com/questions/1854/python-what-os-am-i-running-on for example. That way, people can use any device platform they want.